config/recipes/beats/metricbeat_hosts.yaml

```diff
@@ -0,0 +1,188 @@
+apiVersion: beat.k8s.elastic.co/v1beta1
+kind: Beat
+metadata:
+  name: metricbeat
+spec:
+  type: metricbeat
+  version: 7.8.0
+  elasticsearchRef:
+    name: elasticsearch
+  kibanaRef:
+    name: kibana
+  config:
+    metricbeat:
+      autodiscover:
+        providers:
+        - hints:
+            default_config: {}
+            enabled: "true"
+          host: ${HOSTNAME}
+          type: kubernetes
+      modules:
+      - module: system
+        period: 10s
+        metricsets:
+        - cpu
+        - load
+        - memory
+        - network
+        - process
+        - process_summary
+        process:
+          include_top_n:
+            by_cpu: 5
+            by_memory: 5
+        processes:
+        - .*
+      - module: system
+        period: 1m
+        metricsets:
+        - filesystem
+        - fsstat
+        processors:
+        - drop_event:
+            when:
+              regexp:
+                system:
+                  filesystem:
+                    mount_point: ^/(sys|cgroup|proc|dev|etc|host|lib)($|/)
+      - module: kubernetes
```

Comment:
Not necessary for this PR but I think it would be interesting to have this as a deployment example using autodiscover to scrape the kubelet metrics. There's not really any need to have this as a dset

Reply:
Aren't these scoped to a node? I don't see all nodes/pods when I query kubelet endpoint.

Reply:
Correct, you could autodiscover each node and hit the kubelet on it

Reply:
Hm, we could, but we already have a Metricbeat running everywhere for system metricsets, this would just add another one. I'll create an issue to discuss.